x/irs/keeper/stripping.go

```diff
@@ -148,10 +151,24 @@ func (k Keeper) CalculateRedeemRequiredPtAmount(ctx sdk.Context, pool types.Tran
 	if amountFromStrategy.IsZero() {
 		return sdk.ZeroInt(), types.ErrZeroAmount
 	}
-	ptDenom := types.PtDenom(pool)
-	ptSupply := k.bankKeeper.GetSupply(ctx, ptDenom)
-	requiredPtAmount := ptSupply.Amount.Mul(redeemUt).Quo(amountFromStrategy)
-	return requiredPtAmount, nil
+	supply := k.bankKeeper.GetSupply(ctx, denom)
+	requiredAmount := supply.Amount.Mul(redeemUt).Quo(amountFromStrategy)
```

Comment:
Is it requriedYt amount or requiredUt amount? I think it would be good to clarify the token name on variable

Reply:
requiredYtAmount or requiredPtAmount 
Supports both.